manifests/candlepin.pp

```diff
@@ -106,7 +119,7 @@
       mode   => '0750',
     } ~>
     exec { 'import CA into Candlepin truststore':
-      command => "keytool -import -v -keystore ${amqp_truststore} -storepass ${keystore_password} -alias ${::certs::default_ca_name} -file ${ca_cert} -noprompt",
+      command => "keytool -import -v -keystore ${amqp_truststore} -storepass ${keystore_password} -alias candlepin-ca -file ${ca_cert} -noprompt",
```

Comment:
It'd still be nice to store `candlepin-ca` in a variable, even if it's just a local one in the class. That way we can refer to it in puppet-katello instead of copying it.

Reply:
I don't think we will reference this naming anywhere else.